test/jdk/java/awt/dnd/NoFormatsDropTest/NoFormatsDropTest.java

```diff
@@ -94,17 +94,14 @@ public void init() {
 
         frame.getToolkit().addAWTEventListener(this, AWTEvent.MOUSE_EVENT_MASK);
         frame.pack();
-        frame.setLocationRelativeTo(null);
```

Comment:
It would be better to retain `frame.setLocationRelativeTo(null)` since you are calling `getLocationOnScreen()` later on.

Reply:
I have restored the line